src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {new Excavator(), new Bulldozer(), new Truck()};
+        for (Machine machine: machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
```

Comment:
Consider removing the redundant empty line after the class declaration (line 3) and the main method signature (line 4) to adhere to best practices as per the checklist.

Reply:
Fixed